src/main/java/frc/team5115/Classes/Software/PhotonVision.java

```diff
@@ -42,7 +42,17 @@ public class PhotonVision {
      public PhotonVision() {
          // Change the name of your camera here to whatever it is in the PhotonVision UI.
          photonCamera = new PhotonCamera(frc.team5115.Constants.VisionConstants.leftCameraName);
- 
+         photonCamera = new PhotonCamera(frc.team5115.Constants.VisionConstants.rightCameraName);
+         photonCamera = new PhotonCamera(frc.team5115.Constants.VisionConstants.frontCameraName);
+
+
+        var result = photonCamera.getLatestResult();
+        if (result.hasTargets()){
+            var targets = result.getBestTarget();
+            var yaw = targets.getYaw();
+            var pitch = targets.getPitch();
+            var distance = targets.getBestCameraToTarget();
+        }
```

Comment:
This code is javascript, not java.

Reply:
In java they added the var keyword that you can use if the type is implied. I think it's used here because the type names are long. I think it's just preference either way and probably doesn't make a difference to the compiler.

Reply:
And now I know. Thanks for teaching me something new about Java.